tests/nnstreamer_filter_tvm/unittest_filter_tvm.cc

```diff
@@ -29,11 +37,24 @@ _SetFilterProp (GstTensorFilterProperties *prop, const gchar *name, const gchar
   prop->num_models = g_strv_length ((gchar **) models);
 }
 
+/**
+ * @brief internal function to get model filename
+ */
+static void
+_get_model_file (gchar ** model_file)
+{
+  const gchar * root_path = g_getenv ("NNSTREAMER_SOURCE_ROOT_PATH");
+  gchar * model_name = g_strdup_printf("tvm_add_one_%s.so_", ARCH);
+  *model_file = g_build_filename (
+      root_path, "tests", "test_models", "models", model_name, NULL);
+  g_free (model_name);
```

Comment:
```suggestion
  const gchar * root_path = g_getenv ("NNSTREAMER_SOURCE_ROOT_PATH");
  gchar * model_name = g_strdup_printf("tvm_add_one_%s.so_", ARCH);
  
  *model_file = g_build_filename (
      root_path, "tests", "test_models", "models", model_name, NULL);
  g_free (model_name);
```

What happens if root_path is NULL?

Reply:
Then ```model_file == NULL```. @kimjh12  is it safe ?

Reply:
`NNSTREAMER_SOURCE_ROOT_PATH` is set in `testenv` so IMO it's safe as long as the env is passed correctly by meson.build

Reply:
It's an ENVVAR. Don't assume that it will be always available for you. Be ready for the case where it's undefined or invalid. Unittest executables may be executed with different environments; we release unittest binaries as well!

Reply:
That's true. Updated to find current path if env var is not set